### DISCUSSION TITLE
Feature Request: Control zoom / text size with keyboard shortcuts

### DISCUSSION THREAD

**Discussion by pcgeek86** [massCodeIO/massCode#208] [Ideas]
## Summary

As a user of massCode, I would like to be able to increase / decrease the font size of the application, by using de facto standard keyboard shortcuts.

CTRL + ➕ = increase text size / zoom
CTRL + ➖ = decrease text size / zoom

As of massCode version 3.1.0, these keyboard shortcuts exhibit no behavior.

**Comment by antonreshetov** [comment]
Added in #222
